src/components/BreadCrumb/BreadCrumb.vue

```diff
@@ -29,6 +29,7 @@
               exact
               :to="item.to"
               :href="item.href"
+              :disabled="item.disabled"
```

Comment:
Have you tested `to`, `href` and `disabled` when they are undefined in the item object?

Reply:
You bet, it's often undefined and defaulted to false within the Vuetiy Component. Essentially the same as not including the `disabled` prop.

Reply:
Oh, right.

I looked it up, and the default value for `to` and `href` is undefined, but the default value for `disabled` is false, so you should fall back to that (eg, `item.disabled || false`).

https://vuetifyjs.com/en/api/v-breadcrumbs-item/#props